app/src/context/apiCalls.js

```diff
@@ -4,7 +4,7 @@ import { loginFailure, loginStart, loginSuccess } from "./AuthActions.js";
 export const login = async (user, dispatch) => {
   dispatch(loginStart());
   try {
-    const res = await axios.post("auth/login", user);
+    const res = await axios.post("api/auth/login", user);
```

Comment:
was getting 404 without this, maybe we want to add `/api` to the proxy instead though, whatever is fine